Better exception handling.

Matt3o12 opened the pull request:
A exception will now throw when a incorrect email or password is given

Matt3o12 committed:
A exception will now throw when a incorrect email or password is given.
Add unit tests.

juanriaza commented on tests.py:
```diff
@@ -2,16 +2,17 @@
-        email = os.environ.get('MEGAEMAIL') or sys.argv[0]
-        password = os.environ.get('MEGAPASSWORD') or sys.argv[1]
-        self.api_logged = Mega.from_credentials(email, password)
+        self._email = "matteo.kloiber@me.com";
```
Please keep the enviroment vars. Travis-ci populate email&password from the values provided at `.travis.yml`.

juanriaza commented on mega/mega.py:
```diff
@@ -70,6 +70,9 @@ def login_ephemeral(self):
         self._login_common(res, random_password_key)
 
     def _login_common(self, res, password):
+        if (res == -2) or (res == -9):
```
`if res in (-2, -9)`

Matt3o12 commented:
Tranks for your improvements (I forgot to change it). 
I'm going to change those things and create a pull request again, ok?

juanriaza commented:
It's not necessary to create a new PR. Just commit those bits and they will
appear here.
On 5 Feb 2013 06:58, "Matt3o12" notifications@github.com wrote:

> Tranks for your improvements (I forgot to change it).
> I'm going to change those things and create a pull request again, ok?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/juanriaza/python-mega/pull/3#issuecomment-13115802.

Matt3o12 committed:
Types

Matt3o12 commented:
2 days ago, I fixed those issues and you should have become a notification.

Matt3o12 committed:
Bugfixes

- request.post().raw without stream=True might return nothing (http://docs.python-requests.org/en/latest/user/quickstart/#raw-response-content)
- I forgot to change self.assertEquals(MegaIncorrectPasswordException, …) back to self.assertEquals().

Matt3o12 committed:
Unit tests improvements

Improve the runtime for unit test for 35 seconds to 13 seconds (for me).

Matt3o12 committed:
Types...
